packages/ui/src/features/core/editable-table/editable-table-expanded-row.tsx

```diff
@@ -92,18 +93,19 @@ export const EditableTableExpandedRow = ({
                     ))}
                   </TableRow>
                   <TableRow>
-                    {editedRow.dataPoints.map((d, i) => (
+                    {editedRow.dataPoints.map((d, i, arr) => (
                       <EditableTableCell
                         key={'expandedvalues' + i}
                         value={d.value}
+                        isRatingInput={
+                          arr.findIndex(d => d.name.includes('score')) === i
```

Comment:
We should identify the score cell either by adding a "data type" to the EditDataType, e.g., type = numeric|string|star|options or by adding a prop at the top level EditableTable(ratingColumns: string[])

Reply:
Type added